src/app/shared/components/template/components/task-progress-bar/task-progress-bar.component.scss

```diff
@@ -18,12 +18,14 @@
     border-radius: var(--ion-border-radius-small);
     background-color: var(--ion-color-gray-400);
   }
-  &.completed {
+  &.completed,
+  &[data-variant~="green"] {
```

Comment:
nit(non-blocking)
Green as a success color is quite culturally-specific. Instead of using a hardcoded color would it make more sense to expose a theme variable that can override? E.g. `--task-progress-bar-color` which defaults to using the app primary color. Alternatively if really wanting to override as a one-off might be better to allow user to specify via parameter

Reply:
Re-opening this comment - pr changes still shows the `data-variant~="green"` property but I can't see when checked out locally (??). I also see a hardcoded `--ion-color-green` locally - just want to check what you decided on this?

Reply:
Oh wait - it only shows the diff if viewing the outdated code (as comment line no longer exists on latest). So I think it has been addressed (although a little confusing) - could you just add a quick clarification comment?